src/integration/java/com/ibm/eventstreams/connect/mqsource/MQSourceTaskAuthIT.java

```diff
@@ -139,6 +140,38 @@ public void verifyJmsConnClosed() throws Exception {
         stopper.run();
     }
 
+
+
+    @Test
+    public void verifyJmsConnections() throws Exception {
```

Comment:
This test still passes even without your code change. 

For this test to be useful in verifying your fix, it would need to fail without your code change, but pass with it. 

I'll highlight why I think this is below, but in general I think it's helpful to start by writing a test that recreates the bug you are planning to fix. By seeing the test fail, before doing anything else, you can be sure that you've written a test case that recreates the problem. And it gives you a way of knowing whether you have fixed the problem - you know you have a working fix when your test starts passing. 

If you have a test that already passes before you start making code changes, you have no way of realising if (as is the case here) your code change is not going to fix the problem.